--- arch/arm64/boot/dts/vendor/qcom/pmi632.dtsi
+++ arch/arm64/boot/dts/vendor/qcom/pmi632.dtsi
@@ -34,6 +34,14 @@
 	interrupts = <0 103 0>;
 	#address-cells = <1>;
 	#size-cells = <0>;
 
 	pmic-voter {
 		compatible = "qcom,pmic-voter";
 	};
+
+	/* Nonaktifkan JEITA dan izinkan bypass + override pengisian */
+	qcom,disable-jeita = <1>;
+	qcom,jeita-bypass = <1>;
+
+	charge_limit {
+		qcom,jeita-override = <1>;
+	};

 	pmi632_charger: qcom,qpnp-smb5 {
 		compatible = "qcom,qpnp-smb5";
 		reg = <0x68>;
